salt/ufw/etc_ufw/sysctl.conf

```diff
@@ -32,10 +56,42 @@ net/ipv4/conf/default/log_martians=0
 #net/ipv4/tcp_fin_timeout=30
 #net/ipv4/tcp_keepalive_intvl=1800
 
+# Ignore Directed pings
+#net/ipv4/icmp_echo_ignore_all = 1
+
 # Uncomment this to turn off ipv6 autoconfiguration
 #net/ipv6/conf/default/autoconf=1
 #net/ipv6/conf/all/autoconf=1
 
 # Uncomment this to enable ipv6 privacy addressing
 #net/ipv6/conf/default/use_tempaddr=2
 #net/ipv6/conf/all/use_tempaddr=2
+
+# Performance Recommended Settings:
```

Comment:
No, a separate file is not really necessary.

It is more of a matter of titles - commit and PR titles. This PR has "security" in its title, so this change does not really belong here.

Please either:
 - change the PR title to something more generic (like "sysct default improvements") and distinguish security and performance settings in commit messages, or
 - create a separate PR for performance settings

Furthermore, do you have any indication as to why Jessie defaults are inadequate in this case, or any measurable benefit from these settings? I would not want to maintain a different set of settings than the defaults unless it is beneficial.